lib/events.js

```diff
@@ -38,7 +38,7 @@ const {
   String,
   Symbol,
   SymbolFor,
-  SymbolAsyncIterator
+  SymbolAsyncIterator,
```

Comment:
Considering that's the only change in `events.js` after all, we'd better leave it as is.
```suggestion
  SymbolAsyncIterator
```